bundles/org.openhab.binding.electroluxair/src/main/java/org/openhab/binding/electroluxair/internal/api/ElectroluxDeltaAPI.java

```diff
@@ -46,17 +43,25 @@
  */
 @NonNullByDefault
 public class ElectroluxDeltaAPI {
-    private static final String CLIENT_URL = "https://electrolux-wellbeing-client.vercel.app/api/mu52m5PR9X";
-    private static final String SERVICE_URL = "https://api.delta.electrolux.com/api/";
+    private static final String CLIENT_ID = "ElxOneApp";
+    private static final String CLIENT_SECRET = "8UKrsKD7jH9zvTV7rz5HeCLkit67Mmj68FvRVTlYygwJYy4dW6KF2cVLPKeWzUQUd6KJMtTifFf4NkDnjI7ZLdfnwcPtTSNtYvbP7OzEkmQD9IjhMOf5e1zeAQYtt2yN";
+    private static final String X_API_KEY = "2AMqwEV5MqVhTKrRCyYfVF8gmKrd2rAmp7cUsfky";
+
+    private static final String BASE_URL = "https://api.ocp.electrolux.one";
+    private static final String TOKEN_URL = BASE_URL + "/one-account-authorization/api/v1/token";
+    private static final String AUTHENTICATION_URL = BASE_URL + "/one-account-authentication/api/v1/authenticate";
+    private static final String API_URL = BASE_URL + "/appliance/api/v2";
+    private static final String APPLIANCES_URL = API_URL + "/appliances";
+
     private static final String JSON_CONTENT_TYPE = "application/json";
-    private static final String LOGIN = "Users/Login";
     private static final int MAX_RETRIES = 3;
 
     private final Logger logger = LoggerFactory.getLogger(ElectroluxDeltaAPI.class);
     private final Gson gson;
     private final HttpClient httpClient;
     private final ElectroluxAirBridgeConfiguration configuration;
     private String authToken = "";
+    private Instant tokenExpiry = Instant.now();
```

Comment:
Would this be clearer and sufficient?
```suggestion
    private Instant tokenExpiry = Instant.MIN;
```

Reply:
That should work, I'll change.